### PR TITLE
Remove deprecated Lib flex-layout

### PR DESCRIPTION
**Pull request by Fari-cloudflight**
### Description

Remove deprecated Lib flex-layout

#### In MAD Lib:
- quick-list
- stepper

#### In Demo App:

- component-demos\page-layouts
- components\navigation\user-id
- example-components
- card-editable
- data-table-basic
- data-table-id-generator
- data-table-single
- quick-list-basic
- quick-list-compact-basic
- quick-list-extended
- stepper
- table
- full-page-layouts

### CODE REVIEW

**Comment by diglesias-clf** [comment]
Was the package-lock.json version changed? (I cannot see the diff in Github)

File: src/app/full-page-layouts/full-page-layouts.module.ts [hunk 1]
```diff
@@ -48,6 +47,7 @@ import { ExamplePageTwoComponent } from './example-page-two/example-page-two.com
 
 @NgModule({
   imports: [
+    RouterModule,
```
Comment (diglesias-clf) [inline src/app/full-page-layouts/full-page-layouts.module.ts]: I don't understand why this was added

File: src/app/flex-utility.scss [hunk 2]
```diff
@@ -0,0 +1,76 @@
+.flex-row {
```
Comment (diglesias-clf) [inline src/app/flex-utility.scss:1]: The old file fx-utility.scss can be deleted

**Review by diglesias-clf — CHANGES_REQUESTED** [review]

File: projects/material-addons/src/lib/quick-list/quick-list-compact/quick-list-compact.component.ts [hunk 3]
```diff
@@ -4,7 +4,8 @@ import { BaseQuickListComponent, QuickListItem } from '../base-quick-list.compon
 @Component({
   selector: 'mad-quick-list-compact',
   templateUrl: './quick-list-compact.component.html',
-  styleUrls: [],
+  styles: [`.quick-list-row{flex-direction: row; box-sizing: border-box; display: flex}`],
```
Comment (kohies) [inline projects/material-addons/src/lib/quick-list/quick-list-compact/quick-list-compact.component.ts:7]: Ließe sich das nicht auch mithilfe dieser Klassen im flex-utility.scss lösen?

Comment (Fari-cloudflight) [reply projects/material-addons/src/lib/quick-list/quick-list-compact/quick-list-compact.component.ts:7]: flex-utility ist nur in demo app
projects/material-addons/src/lib kann das nicht verwenden

File: src/app/full-page-layouts/full-page-layouts.module.ts (hunk 1 above)
Comment (Fari-cloudflight) [reply src/app/full-page-layouts/full-page-layouts.module.ts]: removed

File: src/app/flex-utility.scss (hunk 2 above)
Comment (Fari-cloudflight) [reply src/app/flex-utility.scss:1]: it is deleted

**Pull request merged**